sonar-ws/src/main/java/org/sonarqube/ws/client/setting/SetRequest.java

```diff
@@ -103,7 +102,7 @@ public Builder setValues(List<String> values) {
       return this;
     }
 
-    public Builder setFieldValues(List<Map<String, String>> fieldValues) {
+    public Builder setFieldValues(List<String> fieldValues) {
```

Comment:
I'm wondering if we should not provide instead a ``setFieldValues(List<Map<String,String>> fieldValues)`` to hide the JSON...
But in fact, I think the goal is to map as close as possible what is done when using HTTP so I think you can forget.